api/src/main/resources/marquez/db/migration/R__2_runs_view.sql

```diff
@@ -0,0 +1,24 @@
+CREATE OR REPLACE VIEW runs_view
```

Comment:
_Minor_: (Similar comment as above) I would be more descriptive with the description and go with:

> **R__1_add_runs_view.sql**